### PR TITLE
chore: Improve #95

### PR DESCRIPTION
**Pull request by tlgimenes**
## What's the purpose of this pull request?
There were some issues with #95. This PR fixes:
1. Bug on ProductSummary indices report
2. Remove event computation outside of critical rendering path
3. Replace request interception to use data in the markup instead

### CODE REVIEW

File: cypress.json [hunk 1]
```diff
@@ -1,5 +1,5 @@
 {
-  "baseUrl": "http://localhost/",
+  "baseUrl": "http://localhost:9000/",
```
Comment (saranicoly) [inline cypress.json:2]: Isn't the default `localhost:8000` ?

File: cypress/integration/analytics.test.js [hunk 2]
```diff
@@ -147,47 +147,29 @@ describe('view_item event', () => {
...
     cy.getById('product-link')
       .first()
+      .within(() => {
+        cy.getById('buy-button').then(($btn) => {
+          skuId = $btn.attr('data-sku')
+        })
+      })
       .click()
       .then(() => {
-        dataLayerHasEvent('select_item')
```
Comment (saranicoly) [inline cypress/integration/analytics.test.js:159]: Why do not keep this test?

**Review by saranicoly — APPROVED** [review]

File: cypress/integration/analytics.test.js (hunk 2 above)
Comment (tlgimenes) [reply cypress/integration/analytics.test.js:159]: it's already taken into account at line 170

File: cypress.json (hunk 1 above)
Comment (tlgimenes) [reply cypress.json:2]: it depends if you are testing with `yarn serve` or `yarn develop`. Anyways, it should be either 9000 or 8000, not like it was before, hehe

Comment (saranicoly) [reply cypress.json:2]: Oh, nice

**Review by saranicoly — APPROVED** [review]

**Pull request merged**